packages/netlify-cms-core/src/components/Collection/CollectionControls.js

```diff
@@ -48,8 +49,13 @@ function CollectionControls({
           filter={filter}
         />
       )}
-      {sortableFields.length > 0 && (
-        <SortControl fields={sortableFields} sort={sort} onSortClick={onSortClick} />
+      {isSearchEnabled && sortableFields.length > 0 && (
```

Comment:
I would prefer not to mix these two concepts (sort and search) and you can already disable sorting via `sortable_fields: []`

Reply:
Hi @erezrokah,

I have removed the conditional rendering of the sorting button based on `isSearchEnabled`

Thanks,

-Brady